libs/core/src/components/pds-popover/stories/pds-popover.stories.js

```diff
@@ -0,0 +1,58 @@
+import { html } from 'lit';
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+export default {
+  argTypes: extractArgTypes('pds-popover'),
+  args: {
+    hasArrow: false,
+    opened: false
+  },
+  component: 'pds-popover',
+  decorators: [withActions],
+  parameters: {
+    actions: {
+      handles: ['pdsPopoverShow', 'pdsPopoverHide'],
+    },
+  },
+  title: 'components/Popover'
+}
+
+const BaseTemplate = (args) => html`
+  <pds-popover component-id="${args.componentId}" has-arrow=${args.hasArrow} opened=${args.opened} placement=${args.placement}>
+    <div slot="content">
+      <p>Pastrami chuck leberkas, swine biltong tail fatback jowl landjaeger ground round strip steak t-bone cow beef. Filet mignon bresaola pastrami, beef salami hamburger short ribs. Strip steak picanha pork loin tri-tip, prosciutto frankfurter turducken cupim sausage corned beef spare ribs sirloin fatback t-bone bacon. Boudin pork chop capicola short ribs flank rump. Fatback andouille chuck, pork venison t-bone bacon cupim turducken brisket sausage pork loin. Shank buffalo sirloin swine shoulder sausage prosciutto fatback.</p>
+      <div>
+        <pds-button variant="secondary">Cancel</pds-button>
+        <pds-button>Get Started</pds-button>
+      </div>
+    </div>
+    <pds-button variant="secondary">Help</pds-button>
+  </pds-popover>`;
+
+const AvatarDropdownTemplate = (args) => html`
```

Comment:
Maybe as a follow up at some point we could change this story to something like `Trigger` or `Trigger Types`? I found Avatar Popover in the sidebar confusing initially and thought it was some type of variant. Not a major blocker, just felt it's worth a mention.